org.testeditor.web.backend.xtext/src/test/java/org/testeditor/web/backend/xtext/index/IndexServiceClientTest.xtend

```diff
@@ -225,19 +270,69 @@ class IndexServiceClientTest {
 			"dropping type-incompatible element (expected eReference type: Grammar; index service provided element of type: Condition).")
 	}
 
+	@Test
+	def void shouldWarnAboutMissingContextRequest() {
+		// given
+		val expected = mock(IEObjectDescription)
```

Comment:
'expected' is misleading, since never actually checked.

Reply:
right, its what the mocked index service is "expected" to return. It makes sense for some test cases, but not so much for others, like this one (changed it in two other test cases, which did not actually assert on this object).